docs/source/tutorial_notebooks/getting_started_2.ipynb

```diff
@@ -0,0 +1,299 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "9337f001-5e7c-4141-a60c-5e99052aee3d",
+   "metadata": {},
+   "source": [
+    "<div style=\"overflow: hidden;\">\n",
+    "    <img src=\"images/DREGS_logo_v2.png\" width=\"300\" style=\"float: left; margin-right: 10px;\">\n",
+    "</div>\n",
+    "\n",
+    "# Getting started with the DESC data registry (part 2)\n",
+    "\n",
+    "Here we continue our getting started tutorial, introducing \"executions\" and \"dependencies\".\n",
+    "\n",
+    "### What we cover in this tutorial\n",
+    "\n",
+    "In this tutorial we will learn how to:\n",
+    "\n",
+    "- Create a new execution and assign datasets to it\n",
+    "- Connect executions through dependencies\n",
+    "\n",
+    "### Before we begin\n",
+    "\n",
+    "If you haven't done so already, check out the [getting setup](https://lsstdesc.org/dataregistry/tutorial_setup.html) page from the documentation if you want to run this tutorial interactively.\n",
+    "\n",
+    "A quick way to check everything is set up correctly is to run the first cell below, which should load the `dataregistry` package, and print the package version."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7ead9b84-4933-4213-93cb-301d79ef1167",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import dataregistry\n",
+    "print(\"Working with dataregistry version:\", dataregistry.__version__)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f48aec2e-2b35-49ed-be76-8818d9e79b2c",
+   "metadata": {},
+   "source": [
+    "## Executions\n",
+    "\n",
+    "The `dataregistry` nomenclature for a grouping of datasets is an \"execution\". For example, when multiple datasets are produced from a DESC pipeline stage, or numerical simulation, say, the parent pipeline stage or numerical simulation would be the \"execution\", and we associate the individual datasets as child members of that execution.\n",
```

Comment:
I don't think this first sentence gives the right flavor.   The phrase "a grouping of datasets" is very broad.  There are various ways to group datasets and various reasons for doing so. Some of them have to do with an execution (as I think of it) and others don't.   I think of an execution  as an abstraction of a particular run of a program (or script, pipeline stage, etc.. Choose your favorite term).   It associates inputs (one group of datasets) with outputs (another group of datasets) and has some characteristics of its own (e.g., the time the run started, configuration file if any, etc.).

Reply:
I've had another go at the wording